packages/bpk-component-tab-new-bubble/src/BpkBubble.module.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use '../../unstable__bpk-mixins/tokens';
+@use '../../unstable__bpk-mixins/borders';
+
+.bpk-bubble {
+  position: relative;
+  display: inline-flex;
+  width: auto;
+  min-width: tokens.bpk-spacing-xxl();
+  max-width: tokens.bpk-spacing-xxl() * 2;
+  height: tokens.$bpk-line-height-base-tight;
+  padding: 0 tokens.bpk-spacing-md();
+  flex-direction: column;
+  justify-content: center;
+  border-radius: tokens.$bpk-border-radius-xs;
+  background-color: tokens.$bpk-status-danger-spot-day;
+  color: tokens.$bpk-color-white;
+  font-family: tokens.$bpk-font-family-larken;
+  font-size: tokens.$bpk-font-size-xs;
+  text-align: center;
+  white-space: nowrap;
+
+  span {
+    // ts-ignore
+    font-weight: tokens.$bpk-font-weight-bold - tokens.$bpk-font-weight-light !important; // stylelint-disable-line declaration-no-important
+  }
```

Comment:
I think if we use a font mixin (From backpack) we can achieve this without !important